code/game/gamemodes/handofgod/_handofgod.dm

```diff
@@ -316,6 +316,18 @@ var/global/list/global_handofgod_structuretypes = list()
 			return 1
 
 
+/mob/camera/god/proc/is_handofgod_myfollowers(A)
```

Comment:
we know that A is a mob, so you can directly write myfollowers(mob/user) or something.